packer/ansible/roles/mongo/tasks/main.yml

```diff
@@ -28,6 +28,15 @@
     state: present
     line: After=network.target cloud-config.service
 
+# Mongo docs recommend disabling --quiet flag in Production
+# https://docs.mongodb.com/manual/reference/configuration-options/#systemLog.quiet
+- name: Remove --quiet mongo command line option
+  lineinfile:
+    dest: /lib/systemd/system/mongod.service
+    regexp: ^ExecStart=/usr/bin/mongod --quiet --config /etc/mongod.conf
```

Comment:
The regexp doesn't need to be this verbose.  Unless you're worried about not getting the quiet.

Reply:
Good point- I can trim it down, but I did want the --quiet in there.  I'll get rid of the --config part.